lib/exfsm.ex

```diff
@@ -179,6 +198,11 @@ defmodule ExFSM.Machine do
   def event_bypasses(state), do:
     event_bypasses(State.handlers(state))
 
+  def event_multitrans(handlers) when is_list(handlers), do:
+    (handlers |> Enum.map(&(&1.event_multitrans)) |> Enum.concat |> Enum.into(%{}))
```

Comment:
This can be compressed into
```suggestion
    Map.new(Enum.flat_map(handlers, &(&1.event_multitrans)))
```

Reply:
We can do it for the `def fsm` and `def event_bypasses` as well